health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/DrawerFieldComposer.js

```
@@ -276,220 +331,226 @@ function DrawerFieldComposer() {
+                      ],
+                }))
+              }
+            />
+          </div>
+        )} */}
+        {showPopup && (
+          <PopUp
+            className={"app-config-pop"}
+            type={"default"}
+            heading={`${t("ADD_API_DETAILS")}`}
+            children={[]}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)